ansible/main.yml

```diff
@@ -14,7 +14,7 @@
     bosh_cli_version: "2.0.48"
     yq_linux_version: "1.14.0"
     terraform_version: "0.11.7"
-    bosh_bootloader_version: "5.11.6"
+    bosh_bootloader_version: "6.3.0"
```

Comment:
Unless you comment out the bosh_bootloader line in the groovy script, this version will get updated next time someone (me) runs it...